common/persistence/src/main/resources/db/migration/V8__alterFederationBatchInfoAddSource.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE federation_batch_info
+  ADD source_system VARCHAR(10) NOT NULL;
```

Comment:
I doubt, that this works. Adding new column with `NOT NULL` but without default!?!

Reply:
True. Will change it